### ISSUE TITLE
Event pickmeup-change doesn't work in flat type

### ISSUE THREAD

**Issue by Porisey** [nazar-pc/PickMeUp#134]
```
pickmeup(element, {flat: true});
element.addEventListener('pickmeup-change', function (e) { console.log(e, e.detail);});
```
That doesn't work.

**Comment by Porisey** [comment]
```
$calendar.each(function () {
  var elem = this;
  pickmeup(elem, {flat: true});
  $(elem).find('.pickmeup').on('pickmeup-change', function (e) {console.log(e, e.detail);});
});
```

That work

**Comment by nazar-pc** [comment]
Yes, events were accidentally fired on instance element, rather on target.
Should be fixed in 3.0.4, which is already released.
